tests/integration/targets/postgresql_publication/tasks/postgresql_publication_initial.yml

```diff
@@ -495,6 +495,144 @@
       that:
       - result.rowcount == 1
 
+  # https://github.com/ansible-collections/community.postgresql/pull/718
+  - name: Create publication for schema in check mode
+    when: postgres_version_resp.stdout is version('15', '>=')
+    <<: *task_parameters
+    postgresql_publication: 
+      <<: *pg_parameters
+      name: test_schema_pub
+      tables_in_schema: '{{ test_schema }}'
+    check_mode: true
+
+  - assert:
+      that:
+      - result is changed
+      - result.queries == ["CREATE PUBLICATION \"test_schema_pub\" FOR TABLES IN SCHEMA \"{{ test_schema }}\""]
+      - result.schemas == []
+    when: postgres_version_resp.stdout is version('15', '>=')
+
+  - name: Check it's not been created
+    <<: *task_parameters
+    when: postgres_version_resp.stdout is version('15', '>=')
+    postgresql_query: 
+      <<: *pg_parameters
+      query: SELECT * FROM pg_publication_namespace
+
+  - assert:
+      that:
+      - result.rowcount == 0
+    when: postgres_version_resp.stdout is version('15', '>=')
+
+  - name: Create publication for schema
+    when: postgres_version_resp.stdout is version('15', '>=')
+    <<: *task_parameters
+    postgresql_publication: 
+      <<: *pg_parameters
+      name: test_schema_pub
+      tables_in_schema: '{{ test_schema }}'
+
+  - assert:
+      that:
+      - result is changed
+      - result.queries == ["CREATE PUBLICATION \"test_schema_pub\" FOR TABLES IN SCHEMA \"{{ test_schema }}\""]
+      - result.schemas == ["acme_schema"]
```

Comment:
why hardcoded 'acme_schema' and not the `{{ test_schema }}` var?
```
- result.schema == [test_schema]
```

Reply:
i had a feeling that using too many vars in a file scope in such cases is a bit overkill in general
but yeah, let's make it consistent. done, PTAL